src/test/files/FreeResourcesOfAutoCloseableInterface.java

```diff
@@ -43,11 +43,11 @@ public void foo2() {
             System.err.println(e.getMessage());
         } finally {
             if (fr) {
-                fr.close();
+                org.close();
```

Comment:
why do you update this variable ?
but checking the code, I think this code doesn't compile because "fr" variable is not visible in finally block ... hum ... we have to check this wrong example code